UserGuide.md

```diff
@@ -739,7 +737,9 @@ Sometimes you need to share state across custom serializers/deserializers ([see
 
 ### Streaming
 
-In addition Gson's object model and data binding, you can use Gson to read from and write to a [stream](https://sites.google.com/site/gson/streaming). You can also combine streaming and object model access to get the best of both approaches.
+In addition Gson's object model and data binding, you can use Gson to read from and write to a stream with the classes [`JsonReader`](https://javadoc.io/doc/com.google.code.gson/gson/latest/com.google.gson/com/google/gson/stream/JsonReader.html) and [`JsonWriter`](https://javadoc.io/doc/com.google.code.gson/gson/latest/com.google.gson/com/google/gson/stream/JsonWriter.html). These classes operate on a JSON document as a sequence of tokens that are traversed in depth-first order. Because the streams operate on one token at a time, they impose minimal memory overhead.
```

Comment:
Text is partially taken from https://web.archive.org/web/20221102163655/https://sites.google.com/site/gson/streaming. In combination with the links to `JsonReader` and `JsonWriter` this should hopefully suffice because their Javadoc also has usage examples.